frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml

```diff
@@ -1,8 +1,14 @@
 name: Understanding the application process
-question: >
-  Do you understand the application process for Digital Outcomes and Specialists&nbsp;5 on the Digital Marketplace?
-hint: >
+question: |
   You can read about how to apply in the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide#how-to-apply" target="_blank" rel="noopener noreferrer">Digital Outcomes and Specialists suppliers’ guide (opens in new tab)</a>.
+
+  By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:
```

Comment:
Is this the correct content? It doesn't quite scan for me 🤔

Reply:
> `By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:`
>
> Is this the correct content? It doesn't quite scan for me :thinking: 

@NoraGDS - any thoughts? Could we shorten this to `Do you understand`?

Reply:
```suggestion
  By starting an application for Digital Outcomes and Specialists 5, do you understand:
```